### PR TITLE
New rule for tmux lock session keybinding

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:

- Add a new rule to configure the key binding for tmux lock-session command

#### Rationale:

- This rule helps support STIG RHEL-08-020040

#### Review Hints:

- `python3  tests/automatus.py template --libvirt qemu:///session rhel87 --datastream build/ssg-rhel8-ds.xml configure_tmux_lock_keybinding`

### CODE REVIEW

File: products/rhel8/profiles/stig.profile [hunk 1]
```diff
@@ -529,6 +529,7 @@ selections:
 
     # RHEL-08-020040
     - configure_tmux_lock_command
+    - configure_tmux_lock_keybinding
```
Comment (jan-cerny) [inline products/rhel8/profiles/stig.profile:532]: I think you will have to update the profile stability tests

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_keybinding/tests/correct.pass.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# platform = Oracle Linux 8,Red Hat Enterprise Linux 8,multi_platform_fedora
```
Comment (jan-cerny) [inline linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_keybinding/tests/correct.pass.sh:2]: Why do the test scenarios contain Oracle Linux 8 if the rule doesn't have OL 8 in prodtype in rule.yml?

File: products/rhel8/profiles/stig.profile (hunk 1 above)
Comment (yuumasato) [reply products/rhel8/profiles/stig.profile:532]: Added by rebase.

File: linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_keybinding/tests/correct.pass.sh (hunk 2 above)
Comment (yuumasato) [reply linux_os/guide/system/accounts/accounts-physical/screen_locking/console_screen_locking/configure_tmux_lock_keybinding/tests/correct.pass.sh]: Copy pasta error, :P

**Review by jan-cerny — APPROVED** [review]
The Automatus CI failed because the rule introduced by this PR isn't available for RHEL 9 according to its prodtype.

**Pull request merged**